pkg/apis/core/v1alpha1/helper/helpers.go

```diff
@@ -67,13 +68,18 @@ func UpdatedCondition(condition v1alpha1.Condition, status v1alpha1.ConditionSta
 		Reason:             reason,
 		Message:            message,
 		LastTransitionTime: condition.LastTransitionTime,
-		LastUpdateTime:     metav1.Now(),
+		LastUpdateTime:     condition.LastUpdateTime,
```

Comment:
I think we still need the last updated time.
With that change the transition time will contain the same information as the transition time.

we should rather have a better check when we set this status in the controller.

Reply:
What do you mean by "With that change the transition time will contain the same information as the transition time"?

With this change the last update time is only changed if something of the condition has changed. Only in that case it makes sense to update the state. Otherwise we would update a status only because the last update time has changed.

Reply:
oh sry was confused. you're right.